scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz

```
@@ -33,12 +33,13 @@ RUN apt-get update; \
 		ninja-build git wget \
 		libbz2-dev zlib1g-dev git curl uuid-dev \
 		pkg-config openjdk-8-jdk liblzma-dev unzip mlton m4; \
-    apt-get install -qy python-pip python-sphinx;
+    apt-get install -qy python3-pip sphinx-doc sphinx-common;
```

Oh, this is actually a good chance to try removing the old version of sphinx from the image again. Basically this: https://github.com/ethereum/solidity/pull/11672#discussion_r671321645 but last time we tried it my PR that fixes doc warnings was not yet merged and we could not do that (https://github.com/ethereum/solidity/pull/11673#issuecomment-882073364).

```suggestion
    apt-get install -qy python3-pip;
```

---

I see that his is actually a different image. But then Sphinx should not be needed in it at all.